skimage/metrics/simple_metrics.py

```diff
@@ -259,3 +263,53 @@ def normalized_mutual_information(image0, image1, *, bins=100):
     H01 = entropy(np.reshape(hist, -1))
 
     return (H0 + H1) / H01
+
+
+def enhancement_measure(image: np.ndarray,
+                        size: int = 3,
+                        eps: float = 1e-6) -> float:
+    """ The image enhancement measure called EME based on [1]_.
+    It is a way of quantifying improvement of the image after enhancement.
+
+    The function uses a sliding window of user-provided size to measure
+    the mean of log of maximal and minimal intensity ratio
+    within the window.
+
+    Parameters
+    ----------
+    image : array
+        Input image of which the quality should be assessed.
+        Can be either 3-channel RGB or 1-channel grayscale.
+    size : int, optional
+        Size of the window.
+    eps : float, optional
+        Parameter to avoid division by zero.
+
+    Returns
+    -------
+    eme : float
+        The number describing image quality.
+
+    References
+    ----------
+    .. [1] Sos S. Agaian,  Karen Panetta, and Artyom M. Grigoryan.
+            "A new measure of image enhancement.",
+            IASTED International Conference on Signal Processing
+            & Communication, Citeseer, 2000,
+
+    Examples
+    --------
+    >>> from skimage.data import camera
+    >>> from skimage.exposure import equalize_hist
+    >>> img = camera()
+    >>> before = enhancement_measure(img)
+    >>> after = enhancement_measure(equalize_hist(img))
+    >>> before < after
+    True
```

Comment:
Other measure functions in this module are applied on and compare two images. As far as I understand, the EME of an image is not useful on its own but rather it's difference to the EME of an improved image. It might be worth thinking about making the function take two images and return a bool or difference between the EME of both images. That way we also prevent users from assigning to much meaning to the absolute value of the measure.